src/layer/BYOCLayer.ts

```diff
@@ -10,6 +10,8 @@ import {
   SHV3_LOCATIONS_ROOT_URL,
   GetMapParams,
   ApiType,
+  GetStatsParams,
+  GetStats,
```

Comment:
I never notice it before... shouldn't this be named just `Stats`? :)

Fftr & merge!

Reply:
Changed the type name to `Stats`